src/modules/pua_reginfo/notify.c

```diff
@@ -417,12 +450,17 @@ int process_body(str notify_body, udomain_t *domain)
 					if(xmlStrcasecmp(params->name, BAD_CAST "unknown-param")
 							!= 0)
 						goto next_param;
-					len += 1 /* ; */
-						   + strlen(xmlGetAttrContentByName(params, "name"));
+					char *param_name = xmlGetAttrContentByName(params, "name");
```

Comment:
char *param_name should be defined at the top

Reply:
done, thanks